packages/grafana-ui/src/components/Combobox/MultiCombobox.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { MultiCombobox, MultiComboboxProps } from './MultiCombobox';
+
+describe('MultiCombobox', () => {
+  it('should render with options', async () => {
+    const options = [
+      { label: 'A', value: 'a' },
+      { label: 'B', value: 'b' },
+      { label: 'C', value: 'c' },
+    ];
+    render(<MultiCombobox options={options} value={[]} onChange={jest.fn()} />);
+    const input = screen.getByRole('combobox');
+    userEvent.click(input);
+    expect(await screen.findByText('A')).toBeInTheDocument();
+    expect(await screen.findByText('B')).toBeInTheDocument();
+    expect(await screen.findByText('C')).toBeInTheDocument();
+  });
+
+  it('should render with value', () => {
+    const options = [
+      { label: 'A', value: 'a' },
+      { label: 'B', value: 'b' },
+      { label: 'C', value: 'c' },
+    ];
+    const { getByText } = render(<MultiCombobox options={options} value={['a']} onChange={jest.fn()} />);
```

Comment:
I think the current recommendation is to call these methods from `screen`, e.g., `screen.getByText`.

Reply:
Something I need to teach copilot ;)